dspace-api/src/main/java/org/dspace/iiif/logger/CacheLogger.java

```diff
@@ -6,7 +6,7 @@
  * http://www.dspace.org/license/
  */
 
-package org.dspace.app.rest.cache;
+package org.dspace.iiif.logger;
```

Comment:
Just wanted to point out that Raf's questions about why this was moved to `iiif.logger` haven't been addressed/answered.  Does this have to do with the new `SherpaCacheLogger` class? It'd be good to have an answer so that we can understand why some cache classes are being moved around.

It also seems really odd to me that a generic `CacheLogger` is under `org.dspace.iif.logger` package... as this class doesn't seem specific to IIIF.
I *do understand* why `CanvasCacheLogger` is likely under a IIIF package, simply because it relates directly to IIIF Canvases.  But, this class movement is unclear to me as well.

Reply:
@tdonohue @Raf-atmire Classes have been moved to org.dspace.iiif logger because we need them in one of packages scanned during spring boot initialization, as configured in ComponentScan annotation of class org.dspace.app.rest.utils.ApplicationConfig

Reply:
Moreover, it is really specific of IIIF is not something "generic" that can be reused by other service and need to stay on the dspace-api layer as it could be needed by script to interact with the caching system (of course not with the basic cache implementation that is bound to a specific JVM but with more sophisticated configuration based on distributed cache mechanism supported by Spring)

Reply:
Thanks for the clarifications @Micheleboychuk and @abollini .

Reply:
I don't dispute the reason for this move, but I do want to suggest that, in the long term, we would be well advised to re-examine the very narrow focus of our uses of things like `@ComponentScan` and `default-autowire-candidates`.